config/backend-config/workspace-config.go

```diff
@@ -44,6 +49,35 @@ func (workspaceConfig *WorkspaceConfig) GetRegulations() (RegulationsT, bool) {
 	}
 }
 
+func replaceConfigWithEnvVariables(response []byte) (updatedRespnse []byte) {
+	configMap := make(map[string]interface{}, 0)
+
+	err := json.Unmarshal(response, &configMap)
+	if err != nil {
+		log.Error("[Workspace-config] Error while parsing request", err, string(response))
+		return response
+	}
+
+	flattenedConfig, err := flatten.Flatten(configMap, "", flatten.DotStyle)
+
+	for configKey, v := range flattenedConfig {
+		reflectType := reflect.TypeOf(v)
+		if reflectType != nil && reflectType.String() == "string" {
+			valString := v.(string)
+			shouldReplace := strings.HasPrefix(strings.TrimSpace(valString), configEnvReplacer)
+			if shouldReplace {
+				envVariable := valString[len(configEnvReplacer):]
+				response, err = sjson.SetBytes(response, configKey, config.GetEnv(envVariable, ""))
```

Comment:
maybe a log here if env is not found, can be helpful in debugging

Reply:
Maybe even panic here and ask them to either unset the `env.` value in webapp or add it in env here? Otherwise it is easy to miss.